### DISCUSSION TITLE
Downloading Blenderbot2

### DISCUSSION THREAD

**Discussion by rvong65** [facebookresearch/ParlAI#4086] [Q&A] [answered]
Is there any way we can download Blenderbot2 without using the interactive command or save it as a pickle, onnx, or pth file?

**Comment by klshuster** [comment]
The interactive command will download the model automatically for you - does this method not suffice?

**Reply by rvong65** [reply]
@klshuster what I am trying to do is to implement Blenderbot2 from scratch or make it into possibly a Pytorch or Tensorflow model without depending on ParlAI. To be honest, I am confused and guidance would be very appreciated!

**Comment by stephenroller** [comment, marked as answer]
You can manually call the corresponding `download` function for the model that you want: https://github.com/facebookresearch/ParlAI/blob/main/parlai/zoo/blenderbot2/blenderbot2_400M.py

**Reply by rvong65** [reply]
What would be the specified data path @stephenroller?

**Reply by klshuster** [reply]
the datapath is the path to which you want to save the model
